relays/substrate-client/src/chain.rs

```diff
@@ -97,13 +65,13 @@ pub trait TransactionSignScheme {
 }
 
 /// Header type used by the chain.
-pub type HeaderOf<C> = <C as Chain>::Header;
+pub type HeaderOf<C> = bp_runtime::HeaderOf<C>;
```

Comment:
Can we not use the types from `bp_runtime` directly instead of adding this extra layer of indirection?